frontends/web/src/components/mobiledatawarning.tsx

```diff
@@ -25,18 +26,15 @@ interface LoadedProps {
 
 type Props = LoadedProps & TranslateProps;
 
-function MobileDataWarning({ usingMobileData, t }: RenderableProps<Props>): JSX.Element | null {
-    if (!usingMobileData) {
-        return null;
-    }
+function MobileDataWarning({ usingMobileData, t }: RenderableProps<Props>) {
     return (
-        <Status dismissable="mobile-data-warning" type="warning">
+        <Status dismissable="mobile-data-warning" type="warning" hidden={!usingMobileData}>
             {t('mobile.usingMobileDataWarning')}
         </Status>
     );
```

Comment:
It is not necessary (anymore?) to return `JSX.Element | null` of a pure function as long as Typescript can understand it.